roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -308,6 +308,8 @@ expand_persistent_volumes: false
 ## When OpenStack is used, Cinder version can be explicitly specified if autodetection fails (Fixed in 1.9: https://github.com/kubernetes/kubernetes/issues/50461)
 # openstack_blockstorage_version: "v1/v2/auto (default)"
 openstack_blockstorage_ignore_volume_az: "{{ volume_cross_zone_attachment | default('false') }}"
+# set max volumes per node (cinder-csi), default not set
+# node_volume_attach_limit: 25
```

Comment:
I believe that you have put 25 as the suggested default because of the limit of Cinder volumes per Nova VMs?
Maybe the default should go here if it is for Cinder CSI: https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes-apps/csi_driver/cinder/defaults/main.yml

Reply:
Me and @Miouge1 already discussed this in this pr: https://github.com/kubernetes-sigs/kubespray/pull/5666#discussion_r384332976

The variable does not have openstack/cinder name since it could be used by other cloud providers.

The 25 volume is for clouds running en openstack version older then rocky since nova was only able to handle 26 volumes (including the root volume) If you run rocky or later this limit is removed. Since this limit is dependent on the release of openstack nova I would prefer to not set a limit :)

Reply:
Ok I see, it can be left without a prefix.
Thanks for the precision :)